step-release-vis/src/app/models/SnapshotInterval.ts

```diff
@@ -0,0 +1,7 @@
+import {Snapshot} from '../models/Data';
+
+export class SnapshotInterval {
```

Comment:
docs? I have no idea what you want to use this class for, it looks off, but since idk the purpose, I can not give you a hint on how to improve yet

Reply:
added doc in new commit. Will use later when mouse moves inside an environment, as I need the functions in Anca's latest PR to do this.